### ISSUE TITLE
Load from multiple labels does not work as expected when  %00 (empty label) is used

### ISSUE THREAD

**Issue by valmol** [microsoft/spring-cloud-azure#655]
Try to find: key=config.message and label=foo
Value with label foo **does not** exist, while values with empty label **does** exist.
Load from multiple labels does not work as expected when  %00 (empty label) is used
It is expected to get a value with empty label 

bootstrap.properties:
```javascript
spring.cloud.azure.appconfiguration.stores[0].label=spring.cloud.azure.appconfiguration.stores[0].label= %00, foo
```

App config:
Key | Value | Cool |
--- | --- | ---
/application/config.message |  Label  | (No Label)
 &nbsp; | Dev Label | dev 
 &nbsp; | Pro Label | pro

Library:
```xml
<dependency>
    <groupId>com.microsoft.azure</groupId>
    <artifactId>spring-cloud-azure-appconfiguration-config</artifactId>
    <version>1.2.2</version>
</dependency>
```

Azure App Configuration for Spring Cloud:
https://github.com/microsoft/spring-cloud-azure/tree/master/spring-cloud-azure-starters/spring-cloud-starter-azure-appconfiguration-config

## Expected Behavior
Get  "Label"

## Current Behavior
Get  "Pro Label"

## Steps to Reproduce (for bugs)
* Create spring boot application: https://docs.microsoft.com/en-us/azure/azure-app-configuration/quickstart-java-spring-app
* Setup App Config using provided table
* Configure list of label values which include empty label value (%00) and just arbitrary label (bootstrap.properties)
* Fetch value in code

## Your Environment
* SDK version: JDK 11

**Comment by mrm9084** [comment]
@valmol thanks for reporting the issue. 

There seems to be a number of issues here. Documentation wise %00 is null which returns all values.

To get what you want your label value should be `spring.cloud.azure.appconfiguration.stores[0].label= , prod`, but that doesn't seem to work in the current version. I am looking into a solution to the issue.

**Comment by mrm9084** [comment]
@valmol This should be fixed in the new release 1.2.4, but is should be either
spring.cloud.azure.appconfiguration.stores[0].label=,foo
or
spring.cloud.azure.appconfiguration.stores[0].label=\0,foo

**Comment by saragluna** [comment]
@valmol Please try with 1.2.5 since we missed something in the release 1.2.4.

**Comment by mrm9084** [comment]
Fixed as a version 1.2.5. Let us know if their are any additional issues.
